relayer/relayer.go

```diff
@@ -88,7 +88,10 @@ func (r *Relayer) Start(ctx context.Context) error {
 					}
 				}
 
-				return err
+				if err != nil {
```

Comment:
Ideally that should also check case above. Then one more time if same error and so on
`if strings.Contains(err.Error(), "InvalidNonce") || strings.Contains(err.Error(), "Expired"`

Reply:
We can add some retries - I don't think I've ever seen it reach an invalid nonce and not recover in the next step though. Point is it's not necessary it'll be too many otherwise the MQ timeout will be reached anyway.

Reply:
Done in [2672c4c](https://github.com/NethermindEth/near-sffl/pull/115/commits/2672c4c2a6c171376d4f8f0a8e8df87e2350cb29)

Reply:
>  Point is it's not necessary it'll be too many otherwise the MQ timeout will be reached anyway.
It will be reached anyway if relayer fails to send. Relayer can't also handle discarding of the message since it doesn't know state of operator. It's business just to send.